tests/ops/fixtures/saas_example_fixtures.py

```diff
@@ -223,3 +224,13 @@ def oauth2_connection_config(db: Session, oauth2_configuration) -> Generator:
     )
     yield connection_config
     connection_config.delete(db)
+
+
+@pytest.fixture(scope="session")
+def saas_config() -> Dict[str, Any]:
+    saas_config: Dict[str, Any] = {}
+    try:
+        saas_config: Dict[str, Any] = load_toml(["saas_config.toml"])
+    except FileNotFoundError as e:
+        logger.warning("saas_config.toml could not be loaded: %s", NotPii(e))
+    return saas_config
```

Comment:
Is there a different way to do this fixture? With `saas_config.toml` in the `.gitignore` people either won't have this file at all, or will have different values. With that any time the fixture is used the test results will be different.

Reply:
We store all of our secrets in Vault and are able to pull them down in the Github Actions or locally by using a personal Vault token. We limit write access to Vault so there is a requirement for devs working on SaaS connectors to be able to define secrets locally until we're able to persist them to Vault. I thought about using environment variables for this but I like being able to define the secrets in a single location and having the fixtures load them automatically. With that as context, I'm open to other suggestions if there is a more preferred/standard approach!

Reply:
My thought was environment variables also, and I can think of a way to make that work with a file. But looking at the way the secrets get loaded I'm not sure that helps because it doesn't look like we ever use environment variables for this? Outside of the test suite how do these values get loaded?

I do understand the problem you are trying to solve for here and wouldn't block it. I'm just tying to think if there is a way to reduce the chance it would pass tests in one place and not another causing confusion.